src/request/import/ImportApi.js

```diff
@@ -86,7 +93,7 @@ class ImportApi extends TopLevelApi {
 
     /**
      * Determine key type and forward user to Passphrase input
-     *
+
      * @param {string} encryptedKeyBase64 - Encrypted KeyPair in base64 format
```

Comment:
Missplaced empty line. Maybe the ones from above the class definition accidentaly moved here? :smile: